### ISSUE TITLE
Captcha Error

### ISSUE THREAD

**Issue by Catzenjaeger** [PHPFusion/PHPFusion#1020]
securimage3/securimage.php
/live/register.php
is_executable(): open_basedir restriction in effect. File(/usr/bin/lame) is not within the allowed path(s): (/var/www/vhosts/aliencollectors.com/:/tmp/)
Line: 1274

**Comment by RobiNN1** [comment]
In .htaccess add this:
`<DirectoryMatch "/var/www/vhosts/aliencollectors.com/">`
 `php_admin_value open_basedir "/var/www/vhosts/aliencollectors.com/:/tmp/:/"`
`</DirectoryMatch>`
http://stackoverflow.com/questions/1846882/open-basedir-restriction-in-effect-file-is-not-within-the-allowed-paths

**Comment by meangczac** [comment]
Should we do this on all servers? or is it just Catz's server?

If yes, we can add that rule in setup.

**Comment by RobiNN1** [comment]
@Catzenjaeger working now captcha or not?

**Comment by Catzenjaeger** [comment]
i have get a google key and use google .. other dosent work (both 2 and 3)
google works !

**Comment by RobiNN1** [comment]
This code does not work ? 
`<DirectoryMatch "/var/www/vhosts/aliencollectors.com/">
php_admin_value open_basedir "/var/www/vhosts/aliencollectors.com/:/tmp/:/"
</DirectoryMatch>`

**Comment by Catzenjaeger** [comment]
ups ... i haven't a htaccess :) so i forget this sorry

**Comment by RobiNN1** [comment]
workring?

**Comment by meangczac** [comment]
@RobiNN1 , the captcha is working for you?

**Comment by RobiNN1** [comment]
No, problem is in session validation

**Comment by karrak1** [comment]
Try to please the securimage 2 or 3, for example when registering accept any code.

**Comment by meangczac** [comment]
I read this in the morning, but I could not check as I found a big piece of a broken puzzle in the Forum script. 

The nature of the problem is --- if you are a Forum Moderator above, the Mood Buttons will post incorrectly due to... HTML source generated like this...

```
<form name='modform' ... >
<form name='moodform'><button>...</button></form>
<form name='moodform'><button>...</button></form>
<form name='moodform'><button>...</button></form>
</form>
```

So my first thought is to Post the mood through $_GET. Then, I scared people tamper URL (Posting through GET gives me a chill when the security issues pops) - (one), and also more SEO rules need to be done (two). I will pull all my hair and get MASSIVE headache IF SEO rule need to be added. Forum is too heavy on SEO rule.

So, come my second thought is to Post to Ajax the Mood buttons. Then I realized, multiple form submits is required. And iMOD check also conflict. I can't leave modform and moodform with either/or scenario. Will be even bigger problem. Nevertheless, I manage to pass through Defender's token validation in a remote file. I leave the backdoor open, no harm. it's just defender::pageHash() function.
https://github.com/php-fusion/PHP-Fusion/commit/5b3b45f8fe87bbbc59e1dd35e64075b953aceff3

Then, finally, by 3pm, I reached a conclusion to remove the modform and use a hidden input. Just the checkboxes's value capturing... I use JS to capture it and copy value to the modform hidden input. Done. Took me a few hundred screen refereshes and by the time I'm done, now it's 6pm.
https://github.com/php-fusion/PHP-Fusion/commit/8a08ea89eb7d298274dfda442030423e739fd0b1

Now, mood form is working great for both Moderator and User. It's a critical fix.

@RobiNN1 by the `No, problem is in session validation` So, you fix or I fix? I still got plenty to do. Leave it open, I'll see to it at a later date. I think since we got secure3, drop the secure2. Less code to maintain.
